src/materials/ComputeMultipleInelasticDamageStress.C

```diff
@@ -0,0 +1,74 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/*                       BlackBear                              */
+/*                                                              */
+/*           (c) 2017 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "ComputeMultipleInelasticDamageStress.h"
+#include "StressUpdateBase.h"
+
+registerMooseObject("BlackBearApp", ComputeMultipleInelasticDamageStress);
+
+InputParameters
+ComputeMultipleInelasticDamageStress::validParams()
+{
+  InputParameters params = ComputeMultipleInelasticStress::validParams();
+  params.addClassDescription("This ComputeMultipleInelasticStress is to be used with "
+                             "DamagePlasticityStressUpdate"); 
+  return params;
+}
+
+ComputeMultipleInelasticDamageStress::ComputeMultipleInelasticDamageStress(
+    const InputParameters & parameters)
+  : ComputeMultipleInelasticStress(parameters),
+    _D(getMaterialProperty<Real>("damage_variable")),
+    _D_old(getMaterialPropertyOld<Real>("damage_variable"))
+{
```

Comment:
Add code like this to verify that `_models` contains only a `DamagePlasticityStressUpdate` model:
```
if (_models.size() != 1)
  paramError("ComputeMultipleInelasticDamageStress currently can only have one model specified in 'inelastic_models'");
  const DamagePlasticityStressUpdate * temp = dynamic_cast<DamagePlasticityStressUpdate *>(_models[0]);
  if (!temp)
    paramError("Model " + _models[0] + " is not a DamagePlasticityStressUpdate object");

Reply:
addressed